src/helpers/safeGetPostsFromContract.ts

```diff
@@ -15,8 +22,8 @@ export default async function (contract: SCPostStorage) {
           } as PostStructOutput)
       )
       .reverse()
-    // Reverse is needed to get the latest posts first
-  } catch {
+  } catch (error) {
+    console.error('Error occurred on getting posts error:', error)
```

Comment:
How about to send it into `handleError('Error occurred on getting posts error:', error)`?